code/modules/mob/living/carbon/examine.dm

```diff
@@ -519,6 +519,16 @@
 	. += "<a href='?src=[REF(src)];hud=m;evaluation=1;examine_time=[world.time]'>\[Medical evaluation\]</a>"
 	. += "<a href='?src=[REF(src)];hud=m;quirk=1;examine_time=[world.time]'>\[See quirks\]</a>"
 
+/// Collects information displayed about src when examined by a user with a medical HUD.
+/mob/living/carbon/human/get_medhud_examine_info(mob/living/user, datum/record/crew/target_record)
```

Comment:
This is the carbon file not the human file

Reply:
right, I'll make that.